### DISCUSSION TITLE
Where is tweepy.asynchronous.AsyncPaginator?

### DISCUSSION THREAD

**Discussion by Bewinxed** [tweepy/tweepy#1981] [Questions] [answered]
In the docs here, it mentions tweepy.asynchronous.AsyncPaginator.

https://docs.tweepy.org/en/latest/v2_pagination.html#tweepy.asynchronous.AsyncPaginator

However, I can't import it, nor do I seem to find it in the code?

**Comment by Harmon758** [comment, marked as answer]
It was added ~4 hours ago, with https://github.com/tweepy/tweepy/commit/33e444a9d13d53ea024ddb3c9da30158a39ea4f6.

Until v4.11 is released, you'll need to install the latest development version from the master branch to use it.
